packages/eas-cli/src/credentials/android/actions/RemoveGoogleServiceAccountKey.ts

```diff
@@ -0,0 +1,47 @@
+import { GoogleServiceAccountKeyFragment } from '../../../graphql/generated';
+import Log from '../../../log';
+import { confirmAsync } from '../../../prompts';
+import { Account } from '../../../user/Account';
+import { Context } from '../../context';
+import { selectGoogleServiceAccountKeyAsync } from '../utils/googleServiceAccountKey';
+
+export class SelectAndRemoveGoogleServiceAccountKey {
+  constructor(private account: Account) {}
+
+  async runAsync(ctx: Context): Promise<void> {
+    const gsaKeyFragments = await ctx.android.getGoogleServiceAccountKeysForAccountAsync(
+      this.account
+    );
+    if (gsaKeyFragments.length === 0) {
+      Log.error("There aren't any Google Service Account Keys associated with your account.");
```

Comment:
Maybe this should be a warning instead? 🤔 
```suggestion
      Log.warn("There aren't any Google Service Account Keys associated with your account.");
```